docs/ops/activation/LogSoftmax_5.md

```diff
@@ -0,0 +1,42 @@
+## LogSoftMax <a name="LogSoftmax"></a> {#openvino_docs_ops_activation_LogSoftmax_5}
+
+**Versioned name**: *LogSoftmax-5*
+
+**Category**: *Activation*
+
+**Short description**: LogSoftmax computes the log of softmax values for the given input.
+
+**Attributes**
+
+* *axis*
+
+  * **Description**: *axis* represents the axis of which the *LogSoftmax* is calculated. *axis* equal 1 is a default value.
+  * **Range of values**: positive integer value
+  * **Type**: int
+  * **Default value**: 1
+  * **Required**: *no*
+
+**Inputs**:
+
+*   **1**: Input tensor with enough number of dimension to be compatible with *axis* attribute. Required.
```

Comment:
Maybe we should list types of input tensor explicitly. I know that this has been a requirement for most of operations in latest opsets.